lib/src/interface/local_platform.dart

```diff
@@ -48,5 +48,5 @@ class LocalPlatform extends Platform {
   String get version => io.Platform.version;
 
   @override
-  bool get ansiSupported => io.Platform.ansiSupported;
+  bool get ansiSupported => io.stdin.supportsAnsiEscapes;
```

Comment:
Since it would be a breaking change, let's hold off and make the breaking change if and when such an API is added to the SDK.